weaviate/collections/classes/config.py

```diff
@@ -1977,16 +1984,20 @@ def inverted_index(
         )
 
     @staticmethod
-    def replication(factor: Optional[int] = None) -> _ReplicationConfigUpdate:
+    def replication(
+        factor: Optional[int] = None, async_enabled: Optional[bool] = None
```

Comment:
can this be enabled after collection creation?

Reply:
Yep, `test_collection_config.py` has an test updating it